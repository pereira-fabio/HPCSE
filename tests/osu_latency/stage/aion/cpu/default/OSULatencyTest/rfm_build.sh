#!/bin/bash

_onerror()
{
    exitcode=$?
    echo "-reframe: command \`$BASH_COMMAND' failed (exit code: $exitcode)"
    exit $exitcode
}

trap _onerror ERR

make -j 1 CC="cc" CXX="CC" FC="ftn" NVCC="nvcc"
