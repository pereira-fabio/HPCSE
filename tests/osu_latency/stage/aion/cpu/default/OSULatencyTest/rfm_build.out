-reframe: command `make -j 1 CC="cc" CXX="CC" FC="ftn" NVCC="nvcc"' failed (exit code: 2)
